gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-|  [stops.txt](#stopstxt) | **Required** | Individual locations where vehicles pick up or drop off passengers. |
+|  [agency.txt](#agencytxt) | **Required** | Transit agencies that provide the data in this dataset. |
+|  [stops.txt](#stopstxt) | **Required** | Locations where vehicles pick up or drop off riders. |
 |  [routes.txt](#routestxt) | **Required** | Transit routes. A route is a group of trips that are displayed to riders as a single service. |
-|  [trips.txt](#tripstxt)  | **Required** | Trips for each route. A trip is a sequence of two or more stops that occurs at specific time. |
-|  [stop_times.txt](#stop_timestxt)  | **Required** | Times that a vehicle arrives at and departs from individual stops for each trip. |
-|  [calendar.txt](#calendartxt)  | **Conditionally required** | Dates for service IDs using a weekly schedule. Specify when service starts and ends, as well as days of the week where service is available. This file is required unless all dates of service are defined in [calendar_dates.txt](#calendar_datestxt). |
-|  [calendar_dates.txt](#calendar_datestxt)  | **Conditionally required** | Exceptions for the service IDs defined in the [calendar.txt](#calendartxt) file. If [calendar.txt](#calendartxt) is omitted, then [calendar_dates.txt](#calendar_datestxt) is required and must contain all dates of service. |
-|  [fare_attributes.txt](#fare_attributestxt)  | Optional | Fare information for a transit organization's routes. |
-|  [fare_rules.txt](#fare_rulestxt)  | Optional | Rules for applying fare information for a transit organization's routes. |
-|  [shapes.txt](#shapestxt)  | Optional | Rules for drawing lines on a map to represent a transit organization's routes. |
+|  [trips.txt](#tripstxt)  | **Required** | Trips for each route. A trip is a sequence of two or more stops that occur during a specific time period. |
+|  [stop_times.txt](#stop_timestxt)  | **Required** | Times that a vehicle arrives at and departs from stops for each trip. |
+|  [calendar.txt](#calendartxt)  | **Conditionally required** | Service dates specified using a weekly schedule with start and end dates. This file is required unless all dates of service are defined in [calendar_dates.txt](#calendar_datestxt). |
+|  [calendar_dates.txt](#calendar_datestxt)  | **Conditionally required** | Exceptions for the services defined in the [calendar.txt](#calendartxt). If [calendar.txt](#calendartxt) is omitted, then [calendar_dates.txt](#calendar_datestxt) is required and must contain all dates of service. |
+|  [fare_attributes.txt](#fare_attributestxt)  | Optional | Fare information for a transit agency's routes. |
+|  [fare_rules.txt](#fare_rulestxt)  | Optional | Fare rules for a transit agency's routes. |
+|  [shapes.txt](#shapestxt)  | Optional | Rules for mapping vehicle travel paths. |
 |  [frequencies.txt](#frequenciestxt)  | Optional | Headway (time between trips) for routes with variable frequency of service. |
```

Comment:
I'm interpreting "variable" to mean that the route starts off with one headway value and at some point changes to another headway value to accommodate demand. In other words buses run every 10 minutes during rush hour and every 20 minutes during other times. In that sense I think "variable" makes sense but perhaps its potentially confusing.

Reply:
It's extra confusing because in some cases headway may not be variable. A route may run every 15 minutes for the whole time it is in service.

Alternative, could use some more word-smithing, but shows the idea: "Headway (time between trips) for routes with consistent headways over operating span(s)."

Reply:
Ick.  Simply describing this file is currently impossible with the differing `exact_times` definitions.

How about:

>Headway (time between trips) for both frequency-based service and a compressed representation of scheduled service

Reply:
How about:
"Headway (time between trips) for headway-based service or a compress representation of fixed schedule service."

This avoids the "frequency-based" term, which, though it might be useful here because of the GTFS file name, is much much less common in the industry than "headway-based".